ListK.md

```diff
@@ -0,0 +1,161 @@
+# Intro
+
+Welcome to the series of videos about functional programming in Kotlin with Arrow. Arrow is a library that is packed with data types and type classes bringing typed FP to Kotlin. In this video, we're going to learn about the List data type and what it's used for.
+
+# Slide 1
+```
+ListK 
+
+ListKW is available in the arrow-data module under the import arrow.data.ListK 
+
+// gradle
+compile ' io.arrow-kt:arrow-data:$arrowVersion' 
+
+import arrow.data.ListK 
+
+```
+ListK is a higher kinded wrapper around the List collection interface and, it's available in the arrow data library.
+
+# Slide 2
+```
+ListK 
+
+ListK can be initialized with the following:
+
+import arrow.*
+import arrow.core.*
+import arrow.data.*
+
+listOf(1, 2, 3).k()
+
+ListK(listOf(1, 2, 3))
+
+ListK.pure(1)
+
+```
+In the following examples, ListK is created from the Kotlin List type with a convenient k() function, using the constructor and invoking the pure method.
+
+# Slide 3
+```
+ListK :: Monoid 
+
+ListK auto derives its Monoid instance.
+
+val list1 = listOf(1, 2, 3).k()
+
+val list2 = ListKW.pure(1)
+
+list1.combineK(list2)
```

Comment:
`combineK` is from `SemigroupK` not `Monoid`. I would add a Slide for `Traverse` which is one of the most important ops. For example if we have a list of independent parallel computations:
```
val a = async { goRemote1() }.k()
val b = async { goRemote1() }.k()
val c = async { goRemote1() }.k()

val parallelOps: List<DeferredK<Result>> = listOf(a, b, c).k()
val joinedResults: DeferredK<ListK<Result>> = parallelOps.traverse(::identity, DeferredK.applicative())
```
We can use traverse to flip around the effects non blocking and end up with a list of results in the future.